python/sglang/srt/utils.py

```diff
@@ -19,6 +19,8 @@
 
 show_time_cost = False
 time_infos = {}
+import torch.distributed as dist
+from PIL import Image
```

Comment:
move this to `decode_video_base64` so we can make `PIL` an optional dependency.
See `load_image` as an example